cmd/operator/main.go

```diff
@@ -35,10 +36,28 @@ import (
 	monitoringv1 "github.com/coreos/prometheus-operator/pkg/client/monitoring/v1"
 	prometheuscontroller "github.com/coreos/prometheus-operator/pkg/prometheus"
 	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+)
+
+const (
+	logLevelAll   = "all"
+	logLevelDebug = "debug"
+	logLevelInfo  = "info"
+	logLevelWarn  = "warn"
+	logLevelError = "error"
+	logLevelNone  = "none"
```

Comment:
Do we really need `All` and `None`? I don't know what should be more verbose than `Debug`. In addition if users want `None` they can simply do `> /dev/null`, right?

Reply:
Just going with what the go kit logger has so offer here. I think the none case is legit as it saves actual function calls internally, and all is easier to digest for users when they really want to make sure they're actually getting all logs.